cf/test/test_read_write.py

```diff
@@ -883,4 +883,4 @@ def test_write_omit_data(self):
     print("Run date:", datetime.datetime.now())
     cf.environment()
     print()
-    unittest.main(module=__file__.split(".")[0], verbosity=2)
+    unittest.main(verbosity=2)
```

Comment:
Sorry I forgot to fix all of these cases last Friday. I'll put up a PR to sort it today!